guides/common/modules/proc_creating-a-local-file-repository.adoc

```diff
@@ -12,38 +12,62 @@ To create a file type repository in a directory on a remote server, see xref:Cre
 
 To create a file type repository in a local directory, complete the following procedure:
 
-. Ensure the Server and {project-client-name} repositories are enabled.
 ifdef::satellite[]
+. Ensure the Utils repositories are enabled.
```

Comment:
Ought this be singular as there is only one Utils repository?

Reply:
Changed to singular and pushed changes, While pushing changes got merge conflicts were resolved. Please take a look at the commit `merge conflicts` `470abf5` that they are not breaking any existing content.